ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/ChatReply.qml

```diff
@@ -20,8 +20,20 @@ Rectangle {
         anchors.right: parent.right
     }
 
+    ChatImage {
```

Comment:
Better to use a Loader, so that we don't load the component for no reason when there is no image. Eg:
 https://github.com/status-im/nim-status-client/blob/master/ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/NormalMessage.qml#L123-L138
Note that the anchors are on the Loader instead.